src/main/java/fi/otavanopisto/kuntaapi/server/discover/EntityUpdater.java

```diff
@@ -89,14 +89,14 @@ public int getTimerWarmup() {
     return 1000 * 60;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2184)